File: app/src/main/java/org/deiverbum/app/model/BiblicalShort.java
package org.deiverbum.app.model;

import static org.deiverbum.app.utils.Constants.TITLE_SHORT_READING;
import static org.deiverbum.app.utils.Utils.LS2;

import android.text.SpannableStringBuilder;

import org.deiverbum.app.utils.Utils;

public class BiblicalShort extends Biblical {
    //public String forma;
    private LHResponsoryShort responsorio;

    public BiblicalShort() {
    }
public SpannableStringBuilder getHeaderLectura() {
        //String s=String.format(new Locale("es"),"%s    %s",TITLE_SHORT_READING,getRefBreve());
        SpannableStringBuilder ssb=new SpannableStringBuilder(Utils.formatTitle(TITLE_SHORT_READING));
        ssb.append("    ");
        ssb.append(Utils.toRed(getRefBreve()));
        return ssb;
    }

    public String getHeaderForRead() {
        return "LECTURA BREVE.";
    }

    public void setResponsorio(LHResponsoryShort responsorio) {
        this.responsorio=responsorio;
    }

    public SpannableStringBuilder getAll() {
        SpannableStringBuilder sb = new SpannableStringBuilder();
        sb.append(getHeaderLectura());
        sb.append(LS2);
        sb.append(getTexto());
        sb.append(LS2);
        sb.append(responsorio.getAll());
        //sb.append(getResponsorio().getAll());
        return sb;
    }


    public SpannableStringBuilder getAllWithHourCheck(int hourId) {
        SpannableStringBuilder sb = new SpannableStringBuilder();
        sb.append(getHeaderLectura());
        sb.append(LS2);
        sb.append(getTexto());
        sb.append(LS2);
        sb.append(responsorio.getAll(hourId));
        //sb.append(getResponsorio().getAll());
        return sb;
    }

    public SpannableStringBuilder getAllForRead() {
        SpannableStringBuilder sb = new SpannableStringBuilder();
        sb.append(getHeaderForRead());
        sb.append(getTexto());
        sb.append(responsorio.getAllForRead());
        //sb.append(getResponsorioForRead());
        return sb;
    }

}

File: app/src/main/java/org/deiverbum/app/model/BiblicalShort.kt
package org.deiverbum.app.model

import android.text.SpannableStringBuilder
import org.deiverbum.app.util.Constants
import org.deiverbum.app.util.Utils

class BiblicalShort : Biblical() {
    //public String forma;
    private var responsorio: LHResponsoryShort? = null
    fun getHeaderLectura(): SpannableStringBuilder {
        //String s=String.format(new Locale("es"),"%s    %s",TITLE_SHORT_READING,getRefBreve());
        val ssb = SpannableStringBuilder(
            Utils.formatTitle(
                Constants.TITLE_SHORT_READING))
        ssb.append("    ")
        ssb.append(Utils.toRed(getRefBreve()))
        return ssb
    }

    fun getHeaderForRead(): String {
        return "LECTURA BREVE."
    }

    fun setResponsorio(responsorio: LHResponsoryShort?) {
        this.responsorio = responsorio
    }

    fun getAllWithHourCheck(hourId: Int): SpannableStringBuilder {
        val sb = SpannableStringBuilder()
        sb.append(getHeaderLectura())
        sb.append(Utils.LS2)
        sb.append(textoSpan)
        sb.append(Utils.LS2)
        sb.append(responsorio!!.getAll(hourId))
        //sb.append(getResponsorio().getAll());
        return sb
    }

    override fun getAllForRead(): SpannableStringBuilder {
        val sb = SpannableStringBuilder()
        sb.append(getHeaderForRead())
        sb.append(textoForRead)
        sb.append(responsorio!!.allForRead)
        //sb.append(getResponsorioForRead());
        return sb
    }
}